libs/git_wrapper/Git_wrapper.ml

```diff
@@ -103,6 +103,8 @@ exception Error of string
  *)
 let _git_diff_lines_re = {|@@ -\d*,?\d* \+(?P<lines>\d*,?\d*) @@|}
 let git_diff_lines_re = Pcre_.regexp _git_diff_lines_re
+let remote_repo_name_pat = {|^http.*\/(.*)\.git$|}
```

Comment:
just remove this intermediate variable, just put the pattern directly below after Pcre_.regexp
(same for the one above by removing the intermediate _git_diff_lines_re)

Reply:
Will do